src/dstack/_internal/core/backends/base/compute.py

```diff
@@ -2,25 +2,50 @@
 import re
 from abc import ABC, abstractmethod
 from functools import lru_cache
-from typing import List, Optional
+from typing import Any, Dict, List, Optional
 
 import git
 import requests
 import yaml
+from pydantic import BaseModel
 
 from dstack._internal import settings
 from dstack._internal.core.models.backends.base import BackendType
+from dstack._internal.core.models.configurations import RegistryAuth
 from dstack._internal.core.models.instances import (
     InstanceOfferWithAvailability,
     LaunchedGatewayInfo,
     LaunchedInstanceInfo,
 )
 from dstack._internal.core.models.runs import Job, Requirements, Run
+from dstack._internal.server.models import ProjectModel, UserModel
+from dstack._internal.server.services.docker import DockerImage
 from dstack._internal.utils.logging import get_logger
 
 logger = get_logger(__name__)
 
 
+class SSHKeys(BaseModel):
```

Comment:
This is no place for common models. Our approach is to place models in dstack/src/dstack/_internal/core/models. E.g. pool.py.

Reply:
I've moved class to the dstack/src/dstack/_internal/core/models/instances.py and renamed to SSHKey